client/src/components/App.jsx

```diff
@@ -41,7 +41,7 @@ export default function App() {
 
   return (
     <div>
-      <Overview product={featureProduct} styles={styles} reviews={reviews} />
```

Comment:
I think we want to be careful pushing up changes in shared files/folders for the sake of quick and easy merging. Maybe discuss tomorrow morning with team